code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -18,22 +18,91 @@
 	display_name = "a pack of Midoris"
 	path = /obj/item/storage/fancy/cigarettes/cigpack_midori
 
+/datum/gear/marijuanapack
+	display_name = "a pack of medical Marijuana"
+	path = /obj/item/storage/fancy/cigarettes/cigpack_med
+
+/datum/gear/cigar
+	display_name = "Havana Cigar"
+	path = /obj/item/clothing/mask/cigarette/cigar/havana
+	cost = 2
+
+/datum/gear/smokingpipe
+	display_name = "smoking pipe"
+	path = /obj/item/clothing/mask/cigarette/pipe
+	cost = 2
```

Comment:
I'm iffy on the above two; cigar is something you have to get from mining, be captain, or have a premium coin for.

the smoking pipe is a maybe; would like @Necaladun thoughts on this one--you normally buy it from the vendor.